plugins/CoreHome/Controller.php

```diff
@@ -319,4 +321,61 @@ public function saveViewDataTableParameters()
 
         ViewDataTableManager::saveViewDataTableParameters($login, $reportId, $parameters, $containerId);
     }
+
+    /**
+     * Return an array of changes provided by plugins
+     *
+     * @return array
+     */
+    public static function getChanges()
```

Comment:
fyi @bx80 this you might want to move into a new class  so it's not in a controller? Not sure if otherwise someone could call it directly on the controller otherwise and there's probably no need to have it in a controller if it shouldn't be accessible through the web anyway.

Could ideally also add a basic test if none is there yet

Reply:
I've moved the method to a new helper class and added a unit test to check that the method is loading values from a fixture generated file and sorting them correctly.